packages/rollup-plugin/src/index.ts

```diff
@@ -101,7 +101,7 @@ export function vanillaExtractPlugin({
 
       return {
         code: output,
-        map: chunkInfo.map ?? null,
+        map: null,
```

Comment:
The `map` property seems to have been removed, I think in rollup v4 (can't find patch notes for it though).

Based of the discussion in https://github.com/vanilla-extract-css/vanilla-extract/pull/706, I don't think this change affects anything as the `null` was merely used to suppress warnings caused by `chunkInfo.map` being `undefined`.